pom.xml

```diff
@@ -233,9 +255,9 @@
                 <version>${javax-activation.version}</version>
             </dependency>
             <dependency>
-                <groupId>log4j</groupId>
-                <artifactId>log4j</artifactId>
-                <version>${log4j.version}</version>
+                <groupId>ch.qos.reload4j</groupId>
+                <artifactId>reload4j</artifactId>
```

Comment:
Another question: are all log4j.xml property files also compatible with reload4j? Is it possible that my logging set up won't work now because of this change?

Reply:
Reload4J has removed support for some loggers in later versions, due to the code for them being in a poor state. This will affect only very rarely used loggers and won't affect this PR as the version of Reload4J used here still has support for them - one of the current test dependencies requires (but doesn't actually use) one of these loggers.